Assets/Scripts/UI/MainMenu.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MainMenu : MonoBehaviour
+{
+    public void OnKlickPlay(string scene)
```

Comment:
https://trello.com/c/HdOqpxY4/3-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%87%D0%B8%D0%BA%D0%B8-ui-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9